internal/services/disbursement_management_service_test.go

```diff
@@ -420,10 +420,16 @@ func Test_DisbursementManagementService_StartDisbursement_success(t *testing.T)
 					}
 					assert.ElementsMatch(t, wantElements, eventData)
 
+					var expectedTopic string
+					if tc.distributionAccount.IsStellar() {
+						expectedTopic = events.PaymentReadyToPayTopic
+					} else {
+						expectedTopic = events.CirclePaymentReadyToPayTopic
+					}
```

Comment:
💡 I suggest using a `switch/case` statement here without default. The reason being we have a check in place called `exhaustive` that would help us by flagging this piece of code when we add a new platform in the future, making it easier to know we need to increment this particular test when new platforms are added.

https://github.com/stellar/stellar-disbursement-platform-backend/blob/4c4bccd14b037277bff0f35991a447c7d4d20751/.github/workflows/ci.yml#L45-L46

Reply:
That's amazing! Didn't know about this check. Thank you

Reply:
addressed in [7e3a652](https://github.com/stellar/stellar-disbursement-platform-backend/pull/366/commits/7e3a6529726351538e3539db5f5e302d3f782e86)